instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/GTPDisodiumSalt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "Guanosine 5'-triphosphate (GTP) sodium salt hydrate.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "GTP, Disodium salt",
+  "preferredOntologyIdentifier": null,
```

Comment:
The [MgATP instance](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/terminologies/molecularEntity/MagnesiumATP.jsonld) states the pubchem references as preferred ID. Could you please add, the one from your description here?
 
Add: https://pubchem.ncbi.nlm.nih.gov/compound/135818278

Reply:
PubChem isn't an ontology, but I've followed your suggestion until we can find a better entry. I can't find the sodium salt of GTP in ChEBI.

Reply:
Yeah, I think that was the logic that @lzehl used for MgATP.